modules/core/src/main/java/org/apache/ignite/internal/processors/query/schema/SchemaIndexCacheVisitorImpl.java

```diff
@@ -68,6 +69,15 @@
     /** Whether to stop the process. */
     private volatile boolean stop;
 
+    static {
+        int parallelism = IgniteSystemProperties.getInteger(INDEX_REBUILDING_PARALLELISM, 0);
+
+        if (parallelism > 0)
+            DFLT_PARALLELISM = Math.min(parallelism, Runtime.getRuntime().availableProcessors());
```

Comment:
Why we override user defined value here? 
There is a good practice to give user some feedback about wrong value or recommend the optimal one.

Reply:
There is a description in javadoc for INDEX_REBUILDING_PARALLELISM, and overriding is mentioned there. Do you suggest to add logging for this overriding?